packages/shared_preferences/shared_preferences_macos/android/gradle.properties

```diff
@@ -0,0 +1,4 @@
+org.gradle.jvmargs=-Xmx1536M
+android.enableR8=true
+android.useAndroidX=true
+android.enableJetifier=true
```

Comment:
These should be removed.

Reply:
done

Reply:
Maybe this still needs to be pushed? I'm still seeing these lines in a refreshed https://github.com/flutter/plugins/pull/2409/files.

Reply:
Never mind, got the changes now.